packages/ui/components/input-file/src/FileHandle.js

```diff
@@ -0,0 +1,84 @@
+import { FileValidation } from './validators.js';
```

Comment:
we don't use the convention of capitalizing constant names elsewhere in repo, so it would be nicer if we used the same convention (just camelCase) here